tachyon/zk/plonk/circuit/examples/simple_circuit_unittest.cc

```diff
@@ -714,7 +744,8 @@ TEST_F(SimpleCircuitTest, Verify) {
   {
     expected_vanishing_random_poly_commitment = CreateCommitment(
         {"0x0000000000000000000000000000000000000000000000000000000000000001",
-         "0x0000000000000000000000000000000000000000000000000000000000000002"});
+         "0x000000000000000000000000000000000000000000000000000000000000000"
+         "2"});
```

Comment:
```suggestion
         "0x0000000000000000000000000000000000000000000000000000000000000002"});
```
In my PC, this works! I don't think this change is necessary!